dbms/src/Storages/DeltaMerge/RowKeyRange.h

```diff
@@ -84,6 +84,32 @@ struct RowKeyValue
         {
             size_t cursor = 0;
             int_value = DB::DecodeInt64(cursor, *value);
+            if (unlikely(value->size() != sizeof(Int64)))
```

Comment:
I think we should make sure the subfix string is exactly "0x00".

Reply:
Fixed.